src/test/kotlin/com/terraformation/backend/accelerator/db/VoteStoreTest.kt

```diff
@@ -0,0 +1,592 @@
+package com.terraformation.backend.accelerator.db
+
+import com.terraformation.backend.RunsAsUser
+import com.terraformation.backend.TestClock
+import com.terraformation.backend.accelerator.model.VoteModel
+import com.terraformation.backend.db.DatabaseTest
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.accelerator.tables.pojos.ProjectVotesRow
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.mockUser
+import java.time.Instant
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+
+class VoteStoreTest : DatabaseTest(), RunsAsUser {
+  override val user = mockUser()
+
+  private val clock = TestClock()
+  private val store: VoteStore by lazy { VoteStore(clock, dslContext, projectVotesDao) }
+
+  data class VoteKey(val userId: UserId, val projectId: ProjectId, val phase: CohortPhase)
```

Comment:
Might be worth adding this to `EMBEDDABLES` at the bottom of `Config.kt` so jOOQ will generate a class for it and include it as a field in the various table-related classes. (The fields for the individual columns will still be there too.)

Reply:
Yeah, fair point that this is only for the test's internal use. I definitely feel the pain of figuring out naming for test fixtures like this!